assets/inference/environments/minimal-ubuntu22.04-py311-cpu-app-quickstart/context/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM mcr.microsoft.com/azureml/inference-base-2204:{{latest-image-tag}}
+USER root
+WORKDIR /
+ENV AZUREML_CONDA_ENVIRONMENT_PATH=/azureml-envs/minimal
+ENV AZUREML_CONDA_DEFAULT_ENVIRONMENT=$AZUREML_CONDA_ENVIRONMENT_PATH
+
+# Prepend path to AzureML conda environment
+ENV PATH $AZUREML_CONDA_ENVIRONMENT_PATH/bin:$PATH
+
+ENV LD_LIBRARY_PATH $AZUREML_CONDA_ENVIRONMENT_PATH/lib:$LD_LIBRARY_PATH
+
+# Create conda environment
+USER root
+COPY conda_dependencies.yaml .
+RUN conda env create -p $AZUREML_CONDA_ENVIRONMENT_PATH -f conda_dependencies.yaml -q && \
+    rm conda_dependencies.yaml && \
+    conda run -p $AZUREML_CONDA_ENVIRONMENT_PATH pip cache purge && \
+    conda clean -a -y   
+
+RUN chown -R dockeruser:dockeruser /azureml-envs/minimal
+USER dockeruser
+
+COPY . .
+
+# Make port 5001 available to the world outside this container
+EXPOSE 5001
```

Comment:
i didn't see the other dockerfile exposing this port, what did that env do?

Reply:
5001 is exposed by mcr.microsoft.com/azureml/inference-base-2204:20240401.v1
Maybe I don't need to expose it again